site2/docs/how-to-landing.md

```diff
@@ -1,7 +1,7 @@
 ---
 Id: how-to-landing
-title: How-to
-sidebar_label: “”
+title: How to landing
+sidebar_label: "How to landing"
 ---

```

Comment:
I think this is a landing page that intends to use this title. Please revert these changes.

cc @momo-jun @DaveDuggins

Reply:
Yes, it's a landing page for a serial of how-to topics.

Reply:
Does `sidebar_label: “”` need to revert?

Reply:
@labuladong you can change it to `sidebar_label: ""` but actually Docu recognizes both `"` and `“`.

Reply:
😂 Seems `sidebar_label` must be `“”`. if I change it to `""`, error occurs:

![image](https://user-images.githubusercontent.com/37220920/207213254-34c3967e-9e9f-423c-9f0c-c307ffae4525.png)

Reply:
It's a very tricky way to avoid this error. I'm not sure of the reason, I'm reverting it for now.

Reply:
I suggest keeping this field. I guess it's very likely that they will appear in the sidebar soon or later :)

Reply:
I've tested, remove `sidebar_label: “”` works fine. I think removing it is better because using`“”` is too tricky and not a good practice in the document. 

> they will appear in the sidebar soon or later.

We can just add it at any time if we need it.